.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - "node"
```

Comment:
removed this step as it was failing for unknown reason, perhaps incompatible with execution environment?